README.md

```diff
@@ -2,7 +2,6 @@
 
 [![NPM Version][npm-image]][npm-url]
 [![NPM Downloads][downloads-image]][downloads-url]
-[![Build Status][circleci-image]][circleci-url]
```

Comment:
Could we link to GA status added instead?

Reply:
Sure, added worfklow badge